src/TimeService.Business/Helpers/Emails/EmailSender.cs

```diff
@@ -32,6 +32,10 @@ public class EmailSender
         return;
       }
 
+      int day = DateTime.UtcNow.Month == 12
+        ? DateTime.DaysInMonth(DateTime.UtcNow.Year + 1, month: 1)
+        : DateTime.DaysInMonth(DateTime.UtcNow.Year, month: DateTime.UtcNow.Month + 1);

```

Comment:
I guess if you use explicit parameter names for months you should either use it for both year and month or don't use at all

Reply:
done